docs/_rules/xUnit2013.md

```diff
@@ -5,37 +5,51 @@ category: Assertions
 severity: Warning
 ---
 
-# This is a documentation stub
-
-Please submit a PR with updates to the [appropriate file]({{ site.github.repository_url }}/tree/master/docs/{{ page.relative_path }}) or create an [issue](https://github.com/xunit/xunit/issues) if you see this.
-
 ## Cause
 
-A concise-as-possible description of when this rule is violated. If there's a lot to explain, begin with "A violation of this rule occurs when..."
+A violation of this rule occurs when using `Assert.Equals` to check for the specific collection sizes.
```

Comment:
A violation of this rule occurs when `Assert.Equal` or `Assert.NotEqual` is used to check if a collection has 0 or 1 elements.